backend/eth/client.go

```diff
@@ -137,7 +137,7 @@ func (c *Client) getExpiry(
 
 	expiry, err := c.client.GetExpiry(ctx, common.HexToAddress(req.Address))
 	if err != nil {
-		err := errors.New(errors.ErrInternalError, fmt.Errorf("failed to get expiry %s", err.Error()))
+		err := errors.New(errors.ErrInternalError, stderr.Wrapf(err, "failed to get expiry %s", err.Error())
```

Comment:
`err` is already being included as the cause, add context instead: `"failed to get expiry for address %s", req.Address`

Reply:
Good idea, updated `GetPublicKey` and `GetCode` similarly as well. 

The final error message looks something like `[1000] error code InternalError with desc Internal Error. Please check the status of the service. with cause failed to get expiry for address 0x0000000000000000000000000000000000000000: error`